mobify/platform-scaffold
# [Integration Manager] Rename responses to results

This PR renames the ever-confusing `responses.js` files to `results.js`, so that we no longer have to worry about confusing them with HTTP responses.

 **JIRA**: N/A 

## Changes
- Rename all responses.js files to results.js

## How to test-drive this PR
- Build the app
- See that it still runs
